FindTown/FindTown/Presentation/MapScene/AddressSheet/AddressSheetViewController.swift

```diff
@@ -0,0 +1,168 @@
+            titleLabel.topAnchor.constraint(equalTo: self.bottomSheetView.topAnchor, constant: 42),
+            titleLabel.centerXAnchor.constraint(equalTo: self.bottomSheetView.centerXAnchor),
+            
+            backButton.leadingAnchor.constraint(equalTo: self.bottomSheetView.leadingAnchor, constant: 17),
+            backButton.centerYAnchor.constraint(equalTo: self.titleLabel.centerYAnchor),
+            
+            countyCollectionView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 25.0),
+            countyCollectionView.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor, constant: 17.0),
+            countyCollectionView.trailingAnchor.constraint(equalTo: bottomSheetView.trailingAnchor, constant: -17.0),
+            countyCollectionView.bottomAnchor.constraint(equalTo: completeButton.topAnchor, constant: -48.0),
+            
+            villageCollectionView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 32.0),
+            villageCollectionView.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor, constant: 17.0),
+            villageCollectionView.trailingAnchor.constraint(equalTo: bottomSheetView.trailingAnchor, constant: -17.0),
+            villageCollectionView.bottomAnchor.constraint(equalTo: completeButton.topAnchor, constant: screenWidth * -0.128),
+            
+            completeButton.heightAnchor.constraint(equalToConstant: 48),
+            completeButton.leadingAnchor.constraint(equalTo: bottomSheetView.leadingAnchor, constant: 17.0),
+            completeButton.trailingAnchor.constraint(equalTo: bottomSheetView.trailingAnchor, constant: -17.0),
+            completeButton.centerXAnchor.constraint(equalTo: bottomSheetView.centerXAnchor),
+            completeButton.bottomAnchor.constraint(equalTo: bottomSheetView.bottomAnchor, constant: -34.0)
+        ])
+    }
+}
+
+extension Reactive where Base: AddressSheetViewController {
+    
+    var cityType: Binder<CityType> {
+        return Binder(self.base) { (viewController, cityType) in
+            switch cityType {
+            case .county:
+                viewController.villageCollectionView.isHidden = true
+                viewController.countyCollectionView.isHidden = false
+                viewController.backButton.isHidden = true
+                self.completeButtonEnabled.onNext(false)
+            case .village:
+                viewController.villageCollectionView.isHidden = false
+                viewController.countyCollectionView.isHidden = true
+                viewController.backButton.isHidden = false
+            }
+        }
+    }
+    
+    var completeButtonEnabled: Binder<Bool> {
+        return Binder(self.base) { (viewController, isEnabled) in
+            if isEnabled == true {
+                viewController.completeButton.isSelected = true
+            } else {
+                viewController.completeButton.isSelected = false
+            }
+        }
+    }
```

Comment:
```suggestion
    var completeButtonEnabled: Binder<Bool> {
        return Binder(self.base) { (viewController, isEnabled) in
            if isEnabled == true {
                viewController.completeButton.isSelected = true
                viewController.completeButton.isEnabled = true
            } else {
                viewController.completeButton.isSelected = false
                viewController.completeButton.isEnabled = false
            }
        }
    }
```

우선 isEnabled 값도 줘야합니다..!
음.. 호영님께서 제 리뷰에 달아주신걸 봤습니다..!
보통은 isSeleted랑 isEnabled랑 같은 값을 가져서 func으로 빼도 괜찮을 거 같긴한데
다른 값을 가지는 경우도 생길 거 같아서 고민이네요..
예를 들어 버튼 스타일은 isSeleted false 스타일을 가지는데 반대로 선택은 할 수 있는 경우..
func으로 따로 만들고 예외로 이런 상황만 따로 값을 주는 쪽으로 가면 별로일까요..?!

Reply:
func으로 따로 빼는것 좋을 것 같아요! 👍